.genx/package.json

```diff
@@ -3,6 +3,11 @@
   "description": "Genesis Blank App Seed Configuration",
   "version": "3.12.3",
   "license": "Apache-2.0",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "format": "prettier --write \"**/*.js\""
```

Comment:
I'd add the linting step to the CI build, otherwise people will ignore it :) Doesn't have to be done in this PR.
https://github.com/genesiscommunitysuccess/blank-app-seed/blob/main/.github/workflows/build.yml

Reply:
ok, I will update the workflow on a separate PR